q2_feature_table/tests/test_summarize.py

```diff
@@ -516,5 +518,33 @@ def test_vega_spec_nandling(self):
         self.assertEqual(spec['data'][0]['values'], exp)
 
 
+class TabulateSampleFrequencyTests(TestCase):
+
+    def test_basic_case(self):
+        table = biom.Table(np.array([[25, 25, 25], [25, 25, 25]]),
+                           ['O1', 'O2'],
+                           ['S1', 'S2', 'S3'])
+        metadata = tabulate_sample_frequencies(table).to_dataframe()
+
+        self.assertTrue('Frequency' in metadata.columns)
```

Comment:
You can generate the exact DataFrame that you expect as output here, since the input is so simple. Can you do that instead? 

Also, be sure to test for edge cases (e.g, empty feature table either creates empty output, or raises an error - whichever you guys think makes more sense).

Reply:
Ok, yeah this was the one thing I was wondering about. Are there any edge cases here other than "empty?"